app/controllers/application_controller.rb

```diff
@@ -126,4 +126,12 @@ def anonymous_user
   def api_action?
     self.class.to_s.start_with?("Api::")
   end
+
+  def initialize_stripe
+    Rails.configuration.stripe = {
```

Comment:
If I understand the point of this PoC correctly, we probably want to make sure that both of these operations are thread-safe.

Reply:
I decided to just remove this one, because it's not really needed. It's just a value that can be plopped straight into the view.